egs2/chime7_task1/asr1/local/install_dependencies.sh

```diff
@@ -48,4 +47,6 @@ if [ ! -z "$ffmpeg" ]; then
   fi
 fi
 
+pip install git+https://github.com/lhotse-speech/lhotse
```

Comment:
Could you use `python3 -m pip` instead of pip?

These two should be the same usually, but sometimes they can be different. e.g. Overriding PATH to the python2.
ESPnet recommends creating python environment in the instruction, so people sometimes met such a problem.

``python3 -m pip` is the most safe way to specify the python3 for my knowledge.

Reply:
I also changed jiwer dependency too